### PR TITLE
Improve project READMEs

### PR DESCRIPTION
**Pull request by reinink**
This PR improves the READMEs in this project by:

1. Fixing the light/dark mode logo implementation to make it work on NPM.
2. Adding READMEs to the Vue and React libraries. Currently these have no README on NPM.

### CODE REVIEW

File: react/README.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+<p align="center">Beautiful hand-crafted SVG icons, by the makers of Tailwind CSS.<p>
+
+<p align="center">
+  <a href="https://heroicons.com"><strong>Browse at Heroicons.com &rarr;</strong></a>
+</p>
+
+<p align="center">
+    <a href="https://github.com/tailwindlabs/heroicons/releases"><img src="https://img.shields.io/npm/v/heroicons" alt="Latest Release"></a>
+    <a href="https://github.com/tailwindlabs/heroicons/blob/master/LICENSE"><img src="https://img.shields.io/npm/l/heroicons.svg" alt="License"></a>
```
Comment (thecrypticace) [inline react/README.md:19]: these should point to the `@heroicons/react` package

File: vue/README.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+<p align="center">Beautiful hand-crafted SVG icons, by the makers of Tailwind CSS.<p>
+
+<p align="center">
+  <a href="https://heroicons.com"><strong>Browse at Heroicons.com &rarr;</strong></a>
+</p>
+
+<p align="center">
+    <a href="https://github.com/tailwindlabs/heroicons/releases"><img src="https://img.shields.io/npm/v/heroicons" alt="Latest Release"></a>
+    <a href="https://github.com/tailwindlabs/heroicons/blob/master/LICENSE"><img src="https://img.shields.io/npm/l/heroicons.svg" alt="License"></a>
```
Comment (thecrypticace) [inline vue/README.md:19]: these should point to the `@heroicons/vue` package

**Review by thecrypticace — CHANGES_REQUESTED** [review]

File: react/README.md (hunk 1 above)
Comment (reinink) [reply react/README.md]: Good catch! 🤙

File: vue/README.md (hunk 2 above)
Comment (reinink) [reply vue/README.md]: Updated! 👍

**Pull request merged**